baton-maven-plugin/src/main/java/org/technologybrewery/baton/BatonMojo.java

```diff
@@ -132,23 +136,40 @@ public void execute() throws MojoExecutionException, MojoFailureException {
     BatonExecutionSummary performMigration(Collection<MigrationTarget> targets) {
         BatonExecutionSummary executionSummary = new BatonExecutionSummary();
         for (MigrationTarget target : targets) {
-            try {
-                getLog().debug(String.format("Executing Migration: %s (%s)", target.getName(), target.getImplementation()));
-                Class<Migration> implementationClass = (Class<Migration>) Class.forName(target.getImplementation());
-                Constructor<Migration> constructor = implementationClass.getConstructor();
-                Migration migration = constructor.newInstance();
-                FileSet[] migrationSpecificFileSets = (CollectionUtils.isNotEmpty(target.getFileSets())) ? getFileSetsForTarget(target) : fileSets;
-                MigrationSummary migrationSummary = migration.execute(migrationSpecificFileSets);
-                executionSummary.addMigrationSummary(migrationSummary);
-
-            } catch (Exception e) {
-                throw new BatonException("Could not complete migrations!", e);
+            if (isActive(target)) {
+                try {
+                    getLog().debug(String.format("Executing Migration: %s (%s)", target.getName(), target.getImplementation()));
+                    Class<Migration> implementationClass = (Class<Migration>) Class.forName(target.getImplementation());
+                    Constructor<Migration> constructor = implementationClass.getConstructor();
+                    Migration migration = constructor.newInstance();
```

Comment:
I: realized that name and description were never naming it to the Migration instance